client/reader/discover/components/tags-navigation/index.tsx

```diff
@@ -30,18 +30,8 @@ export const useRecommendedTags = (): Tag[] => {
 	const { data: interestTags = [] } = useQuery< InterestResponse, unknown, Tag[] >( {
 		queryKey: [ 'read/interests', locale ],
 		queryFn: () =>
-			wpcom.req.get(
-				{
-					path: `/read/interests`,
-					apiNamespace: 'wpcom/v2',
-				},
-				{
-					_locale: locale,
-				}
-			),
-		select: ( data ) => {
-			return data.interests;
-		},
+			wpcom.req.get( { path: '/read/interests', apiNamespace: 'wpcom/v2' }, { _locale: locale } ),
```

Comment:
I see some files where the change is only a code style change rather than a change related to this PR, like the change here. This is increasing the number of files involved in this PR.

Was the change in this file necessary?

I am asking because a PR that refactors/touches how we build API requests can be a risky one, and increasing the number of changed files to this PR can add more noise when trying to debug an issue that might come up after deploying it.

Reply:
You're right, the PR got little out of hand, and has changes beyond the original intent, which was to remove trailing slashes from `apiNamespace` arguments. Now there's also `apiVersion` removal, converting backtick to regular strings... I'll try to do better next time.